client-common/src/tendermint/rpc_client.rs

```diff
@@ -65,4 +65,22 @@ impl Client for RpcClient {
         self.call::<serde_json::Value>("broadcast_tx_sync", &params)
             .map(|_| ())
     }
+
+    fn query(&self, path: &str, data: &str) -> Result<QueryResult> {
+        // path, data, height, prove
+        let params = [json!(path), json!(data), json!(null), json!(null)];
+        self.call("abci_query", &params)
+    }
+    fn get_account(&self, staked_state_address: &[u8]) -> Result<StakedState> {
+        self.query("account", hex::encode(staked_state_address).as_str())
```

Comment:
As mentioned above, I don't think this implementation belongs in `RpcClient`. We can move all the custom login in `NetworkOpsClient` implementation. Any thoughts @tomtau?